package.json

```diff
@@ -80,6 +80,7 @@
     "dotenv": "16.4.5",
     "eta": "2.2.0",
     "extract-zip": "2.0.1",
+    "fast-json-stringify": "5.14.1",
```

Comment:
So is it best to use this? Or only ajv?

Reply:
I didn't found advantages to use one instead of the other, so I continued to use fast-json because it is what fastify are using for serialization (https://fastify.dev/docs/latest/Reference/Validation-and-Serialization/#serialization).